app/docs/src/tutorials/generate-a-workspace-api-token.md

```diff
@@ -0,0 +1,46 @@
+# Creating an API Token for your workspace
+
+This tutorial will teach you how to create an API token for your workspace.
+
+To follow along, you should:
+
+1. Finish the [Getting Started tutorial](./getting-started), to have basic
+   knowledge of System Initiative.
+
+2. Have your System Initiative workspace open.
+
+
+This tutorial shows you how to create an API token scoped to your current workspace. This can be used to interact with the System Initiative public API.
+
+## Create an API token for your workspace
+
+Click the Cog icon in the top right corner of the workspace, then select `Manage Workspace Tokens`.
+
+![Manage Workspace Tokens](./generate-a-workspace-api-token/manage-workspace-tokens.png)
+
+Enter a `Token Name` and a value in the `Expiration` field, and click `Generate API Token`.
```

Comment:
You will want to say what the expiration values are - e.g. 1d for 1day, 1w for 7 days etc.